pom.xml

```diff
@@ -88,34 +91,99 @@
         <dependency>
             <groupId>joda-time</groupId>
             <artifactId>joda-time</artifactId>
-            <version>2.10.13</version>
+            <version>2.12.2</version>
         </dependency>
 
         <dependency>
             <groupId>com.github.onsdigital</groupId>
             <artifactId>dp-logging</artifactId>
             <version>${dp.logging.version}</version>
         </dependency>
-        <dependency>
-            <groupId>com.github.onsdigital</groupId>
-            <artifactId>dp-jwt-verifier-java</artifactId>
-            <version>0.5.0</version>
-        </dependency>
+
     </dependencies>
 
     <build>
+        <pluginManagement>
```

Comment:
I don't think we need to use `pluginManagement` as we don't have projects inheriting from this one? Could we just use `plugins`?

Reply:
ref :- 
https://github.com/ONSdigital/dp-jwt-verifier-java/blob/main/pom.xml 
and 
https://www.baeldung.com/maven-plugin-management